iOS/Controllers/HomeworkList/HomeworkListCourseSortedViewController.swift

```diff
@@ -30,12 +30,11 @@ final class HomeworkListCourseSortedViewController: UITableViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        let nib = UINib(nibName: "HomeworkHeaderView", bundle: nil)
-        self.tableView.register(nib, forHeaderFooterViewReuseIdentifier: "HomeworkHeaderView")
-
+        self.tableView.register(UINib(resource: R.nib.homeworkHeaderView),
```

Comment:
Please use `registerHeaderFooterView` for this. The `homeworkHeaderView` will need an identifier in order to work correctly.